website/thunderbird/102.0/eoy/index.html

```diff
@@ -0,0 +1,109 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+ {% set active_page = "eoy" %}
+ {% extends "_includes/bedrock/base-resp.html" %}
+ {% block facebook_id %}348169218537151{# facebook.com/Thunderbird #}{% endblock %}
+ {% block twitter_id %}mozthunderbird{% endblock %}
+
+ {% block page_image %}{{ static('img/thunderbird/thunderbird-256.png') }}{% endblock %}
+ {% block page_favicon %}{{ static('img/thunderbird/favicon.ico') }}{% endblock %}
+ {% block page_favicon_large %}{{ static('img/thunderbird/favicon-196.png') }}{% endblock %}
+ {% block page_ios_icon %}{{ static('img/thunderbird/ios-icon-180.png') }}{% endblock %}
+
+ {% block page_css %}<link href="/media/css/eoy.css" rel="stylesheet" type="text/css" />{% endblock %}
+
+ {% block page_title %}{{ _('Help Keep Thunderbird Alive!') }}{% endblock %}
+
+ {% block site_header %}
+ <div id="forest-bg">
+
+</div>
+<header>
+  {% block header_content %}
+  <section id="initAppeal" class="text-left font-xl pl-8 pr-8 md:pl-16 md:pr-16 max-w-8xl mx-auto pt-12">
+    <aside>
+      <h1 class="accent-text">
+        <span class="font-light">{{ _('Help Keep')}}</span>
+        <br>
+        <span>{{ _('Thunderbird Alive!')}}</span>
```

Comment:
I spoke with Andrei and indeed this would not be understandable or usable for a lot of languages.

Also I would suggest to style the whole H1 with the larger font we need for the "Thunderbird Alive" part, and then a simple span without class to style the exception, which is the smaller and thinner font.

We need to have something like this, I think:
```
{{ _('<span>Help Keep</span>Thunderbird Alive!')}}
```

We can also avoid the `<br>` by making that span a display block.